src/services/github.js

```diff
@@ -0,0 +1,256 @@
+import { getNetworkName } from '@utils/web3-utils'
+import env from '@/environment'
+
+const NETWORK = getNetworkName().toLowerCase()
+const GITHUB_API_TOKEN = env('GITHUB_API_TOKEN')
+
+export const fetchLatestCommitSha = async () => {
+  const endpoint =
```

Comment:
Wondering if it makes sense to have all endpoints in constants above and have a `ENDPOINT_BASE=https://api.github.com/repos/1Hive/dao-list-test` and derive all endpoints from this one?